openslides_backend/action/actions/resource/mixins.py

```diff
@@ -21,5 +21,6 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
         ):
             return
 
-        msg = "Missing can_manage_organisation"
+        msg = f"You are not allowed to perform action {self.name}."
+        msg += "Missing can_manage_organisation"
         raise PermissionDenied(msg)
```

Comment:
Please enhance `MissingPermission` so that it also takes OML/CML as input

Reply:
What is OML and CML?

Reply:
Organisation management level and committee management level - see https://github.com/OpenSlides/OpenSlides/wiki/Permission-System#organisation-management-level-oml

Reply:
We should also allow string and other stuff here. So I use Any as type.

Reply:
Where do you want to give strings? I would suggest that `Union[Permission, OrganisationManagementLevel, CommitteeManagementLevel]` should be what we want

Reply:
Done.

Reply:
Oh, not done, I run into circular imports with it.
* permissions.management_level import exceptions.PermissionException
* exceptions import permissions.management_level.
I think `Any `is a good solution to this.

Reply:
Using `Any` is the lazy solution when deaaling with circular imports :D I think the mistake here is that the management levels throw a `PermissionDenied`, this should be a simple `TypeError`.

Reply:
Changed.